get-influxdb-data.sh

```diff
@@ -44,13 +44,14 @@ typeset -r INFLUXDB_QUERY_WHERE_DFLT='time > now() - 1d'
 typeset -r INFLUXDB_QUERY_GROUP_DFLT='time(1ms), "displayName"'
 typeset -r INFLUXDB_QUERY_FILL_DFLT="none"
 typeset -i DAYS_DFLT=1
+typeset -r OUTPUTFILE_DFLT="/dev/stdout"
 
 #### argument scanning:  usage ####
-typeset -r USAGE="${PNAME} -[Dhpv d* f* g* q* S* s* t* u* w*]"
+typeset -r USAGE="${PNAME} -[Dhpv d* f* g* o* q* S* s* t* u* w*]"
 
 # produce the help message.
 function _help {
-	more 1>&2 <<.
+	less 1>&2 <<.
```

Comment:
Need use `$MORE` (for example), and set MORE to a suitable pager. This requires additional setup in the head matter.